### PR TITLE
Pet collection integration

### PR DESCRIPTION
**Pull request by mlima95**
[ADD] - Pet collections to select companion scene

### CODE REVIEW

**Review by moufmouf — CHANGES_REQUESTED** [review]
One thing seems broken. When I come back to the select companion scene after selecting a companion, the view does not open on the companion I selected. It selects the first companion of the list by default.

**Comment by mlima95** [comment]
> One thing seems broken. When I come back to the select companion scene after selecting a companion, the view does not open on the companion I selected. It selects the first companion of the list by default.
@moufmouf -> cc69bebb02d8d38b8614437c35f4635d6da021a3 f6e82fbe3715fc09c7cfd5b6b8b7ee24e52b38d2

File: play/src/front/Components/SelectCompanion/SelectCompanionScene.svelte [hunk 1]
```diff
@@ -23,11 +24,35 @@
...
     }
+
+    function selectLeftCollection() {
+        selectCompanionScene.selectPreviousCompanionCollection();
+    }
+
+    function selectRightCollection() {
+        selectCompanionScene.selectNextCompanionCollection();
+    }
+    console.log(selectedCollection);
```
Comment (moufmouf) [inline play/src/front/Components/SelectCompanion/SelectCompanionScene.svelte:35]:
```suggestion
```

File: play/src/pusher/services/QueryValidator.ts [hunk 2]
```diff
@@ -35,7 +35,12 @@ export function validateWebsocketQuery<T extends ZodObject<ZodRawShape>>(
     validator: T
 ): z.infer<T> | undefined {
     const urlSearchParams = new URLSearchParams(req.getQuery());
-    const params = Object.fromEntries(urlSearchParams.entries());
+    /* eslint-disable @typescript-eslint/no-explicit-any */
+    const params: any = {};
```
Comment (moufmouf) [inline play/src/pusher/services/QueryValidator.ts:39]: Can we try this?

```suggestion
    const params: Record<string, string|string[]> = {};
```

**Review by moufmouf — APPROVED** [review]

**Pull request merged**